activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Add `rake db:prepare`  runs setup if database does not exist, or runs migrations if it does
```

Comment:
nitpick: _I think it is common to add entry as a sentence, to make this sentence I think it should end with **.**_ symbol.